### DISCUSSION TITLE
post/istio-series-part-1-setup-and-installation/

### DISCUSSION THREAD

**Discussion by giscus** [giovanism/e#10] [Announcements]
# post/istio-series-part-1-setup-and-installation/

Hi, long time no see! I just started learning Istio seriously so I would like a
place to write down my understanding and additional thoughts I have.

https://deploy-preview-9--giovanisme.netlify.app/post/istio-series-part-1-setup-and-installation/

**Comment by giovanism** [comment]
Testing giscus anjay
